src/Workspaces/Remote/Core/BrokeredServiceConnection.cs

```diff
@@ -184,6 +184,11 @@ private static bool IsRemoteIOException(Exception exception)
 
         private void OnUnexpectedException(Exception exception, CancellationToken cancellationToken)
         {
+#if DEBUG
+            // Treat unexpected exceptions as failures during debug builds
+            FatalError.Report(exception);
+#endif
```

Comment:
📝 This change will end up getting reverted, but currently there is no way to see why all brokered services are always failing:

![image](https://user-images.githubusercontent.com/1408396/93735757-e68cb700-fb92-11ea-86a0-bc7faa47c442.png)

Reply:
I finally fixed this issue via 840bcbc5e6882d8984b7b91489c94b560d409efb